hilti/toolchain/src/compiler/driver.cc

```diff
@@ -891,9 +889,13 @@ Result<Nothing> Driver::initRuntime() {
         hookInitRuntime();
     } catch ( const hilti::rt::Exception& e ) {
         printHiltiException(e);
+        hookFinishRuntime();
+        rt::done();
```

Comment:
Just as a note, should we ever require explicit teardown for the Spicy runtime, we wouldn't be able to do it before this `exit` (since we are in HILTI), and we'll probably need to use `_exit` here as well. This should work now though.

Similarly below.

Reply:
The `hookFinishRuntime()` takes care of that: that executes the Spicy shutdown.